ios/LEGACY/Fabric/LEGACY_RNCPagerViewComponentView.mm

```diff
@@ -309,6 +309,10 @@ - (BOOL)isLtrLayout {
     return [_layoutDirection isEqualToString: @"ltr"];
 }
 
+- (BOOL)isHorizontalRtlLayout {
```

Comment:
Could you extract this function to a separate file? It is used in two places.

Reply:
Good point. I've done this in separate commit, as it introduces some boilerplate obj-c code. I'm more of an expert in swift than obj-c, so please verify.